lib/ExpensiMark.js

```diff
@@ -47,7 +47,7 @@ export default class ExpensiMark {
 
                 // Use the url escaped version of a backtick (`) symbol. Mobile platforms do not support lookbehinds,
                 // so capture the first and third group and place them in the replacement.
```

Comment:
Could you also please update the comment to reflect the change for quick readers